apps/pie-docs/src/components/tag/overview/overview.md

```diff
@@ -44,315 +47,332 @@ Tags can be embedded in other components such as cards, data tables (among other
+        "**Label:** Provides informative information to the user.",
+        "**Trailing icon (Optional):** Indicates additional actions or further interactions."
     ]
 } %}
 
 ---
 
 ## Variations
 
-### Type
+### Non-interactive 
 
-Tag can use colour for visual categorisation.
+The non-interactive variation should be used by default, and should be used when the tag doesn’t require any interactive features tied to the instance.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/tag/non-interactive-tag.svg",
+    alt: "A tag with a leading icon placeholder and a label text",
+    width: "69px"
+} %}
+
+### Interactive
+
+The interactive variation should be used for non-crucial interactions.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/tag/interactive-tag.svg",
+    alt: "Different states of the interactive tag component such as default, hover and active states",
+    width: "224px"
+} %}
+
+--- 
+
+## Modifiers
+
+### Icon
+
+When incorporating an icon into a tag, it is essential to ensure that the icon clearly supports the label. Icons are only available in the large size of the tag. 
 
 {% contentLayout %}
   {% contentItem %}
-    <h3>Neutral</h3>
+    <h4>Leading</h4>
+    <p>Leading icon provides context and visually supports the label and it is available for both variants.</p>
     {% contentPageImage {
-      src: "../../../assets/img/components/tag/variation-type-neutral.svg",
-      width: "106px",
-      alt: "A pair of strong and subtle neutral tags."
+      src: "../../../assets/img/components/tag/modifier-icon-leading.svg",
+      width: "69px",
+      alt: "A tag component with a leading icon and a label text"
     } %}
   {% endcontentItem %}
   {% contentItem %}
-    <h3>Blue</h3>
+    <h4>Trailing</h4>
+    <p>Trailing icon offers additional actions or indicates further interactions, for this reason it is only available in the interactive set.</p>
     {% contentPageImage {
-      src: "../../../assets/img/components/tag/variation-type-blue.svg",
-      width: "106px",
-      alt: "A pair of strong and subtle tags in blue color."
+      src: "../../../assets/img/components/tag/modifier-icon-trailing.svg",
+      width: "69px",
+      alt: "A tag component with a trailing icon and a label text"
     } %}
   {% endcontentItem %}
+{% endcontentLayout %}
+
+### Emphasis
+
+Depending on the level of visual prominence you want to give to the tag, you can choose between strong or subtle emphasis.
+
+{% contentLayout %}
   {% contentItem %}
-    <h3>Green</h3>
+    <h4>Strong</h4>
     {% contentPageImage {
-      src: "../../../assets/img/components/tag/variation-type-green.svg",
-      width: "106px",
-      alt: "A pair of strong and subtle tags in green color."
+      src: "../../../assets/img/components/tag/modifier-emphasis-strong.svg",
+      width: "69px",
+      alt: "A tag component with the strong modifier to indicate emphasis"
```

Comment:
I don't know if this is very useful alt text. Should it describe the image a bit more?